src/ontology/cl-edit.owl

```diff
@@ -22083,6 +22084,21 @@ AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0015000 "cranial motoneuron"
 AnnotationAssertion(rdfs:label obo:CL_0015000 "cranial motor neuron")
 SubClassOf(obo:CL_0015000 obo:CL_0000100)
 
+# Class: obo:CL_0017000 (pulmonary ionocyte)
+
+AnnotationAssertion(obo:IAO_0000115 obo:CL_0017000 "An ionocyte that is part of the lung epithelium. The cells from this type are major sources of the CFTR protein in human and mice. 
+
+References:
+
+1. Plasschaert, L.W., Žilionis, R., Choo-Wing, R. et al. A single-cell atlas of the airway epithelium reveals the CFTR-rich pulmonary ionocyte. Nature 560, 377–381 (2018).https://doi.org/10.1038/s41586-018-0394-6
+
+2. Montoro, D.T., Haber, A.L., Biton, M. et al. A revised airway epithelial hierarchy includes CFTR-expressing ionocytes. Nature 560, 319–324 (2018). https://doi.org/10.1038/s41586-018-0393-7")
+
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_0017000 <https://orcid.org/0000-0003-2473-2313>)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasOBONamespace> obo:CL_0017000 "cell")
+AnnotationAssertion(rdfs:label obo:CL_0017000 "pulmonary ionocyte")
+SubClassOf(obo:CL_0017000 obo:CL_0000082)
```

Comment:
I was not sure wheter to add it, because the definition of ionocyte in CL does not include pulmonary ionocytes, as this was a newly discovered type of cell. I have added it for now, but I believe that for it to be 100% accurate, maybe the definition of ionocyte would need to be broader in scope.